dts/arm/renesas/ra/ra2/r7fa2a1ab3cfm.dtsi

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2024 TOKITA Hiroshi
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/renesas/ra/ra2/r7fa2a1xh.dtsi>
+
+/ {
+	soc {
+		flash-controller@407fe000 {
```

Comment:
It is incorrect. Address should be "407e0000", could you please recheck?

Reply:
https://www.renesas.com/us/en/document/mah/renesas-ra4m1-group-users-manual-hardware?r=1054146#G8.2206429

You're correct. Thank you for your pointing out.
Fixed it.